ci/purge_expired_sessions_from_datastore.yaml

```diff
@@ -0,0 +1,44 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  PROJECT_ID:
+  DATAFLOW_TEMPLATE_VERSION:
+  EXPIRATION_TIME_OFFSET:
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=/root/gcloud-service-key.json
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      gcloud auth activate-service-account --key-file "${GOOGLE_APPLICATION_CREDENTIALS}"
+      gcloud config set project "${PROJECT_ID}"
+
+      DATAFLOW_JOB=$(gcloud dataflow jobs run purgesession \
+      --region europe-west2 \
+      --gcs-location gs://dataflow-templates/"${DATAFLOW_TEMPLATE_VERSION}"/Datastore_to_Datastore_Delete \
+      --parameters \
+      datastoreReadGqlQuery="SELECT * FROM \`eq-session\` WHERE expires_at < $((`date --utc +%s` - EXPIRATION_TIME_OFFSET)) ",\
+      datastoreReadProjectId="${PROJECT_ID}",\
+      datastoreDeleteProjectId="${PROJECT_ID}" \
+      --format="value(id.scope())")
```

Comment:
Can the formatting be updated to make it more readable?.

Reply:
Bash is never pretty, and there is not much I can do here